src/main/java/uk/gov/hmcts/reform/civil/service/PaymentsService.java

```diff
@@ -58,20 +81,14 @@ private CreditAccountPaymentRequest buildRequest(CaseData caseData) {
 
         String customerReference = ofNullable(generalAppPBADetails.getPaymentDetails())
             .map(PaymentDetails::getCustomerReference)
-            .orElse(generalAppPBADetails.getPbaReference());
+            .orElse(generalAppPBADetails.getServiceReqReference());
 
-        return CreditAccountPaymentRequest.builder()
+        return ServiceRequestPaymentDto.builder()
             .accountNumber(generalAppPBADetails.getApplicantsPbaAccounts()
                     .getValue().getLabel())
             .amount(claimFee.getCalculatedAmount())
-            .caseReference(caseData.getLegacyCaseReference())
-            .ccdCaseNumber(caseData.getCcdCaseReference().toString())
             .customerReference(customerReference)
-            .description("Claim issue payment")
             .organisationName(organisationName)
-            .service(paymentsConfiguration.getService())
-            .siteId(paymentsConfiguration.getSiteId())
```

Comment:
payments configuration site id is what we need to pass in the org id of the service request

Reply:
Done changed and committed